zuul.d/adoption.yaml

```diff
@@ -125,9 +130,14 @@
       - OWNERS
       - .*/*.md
 
+# TODO(marios): remove this once we finish rename OSPRH-8452
 - job:
     name: cifmw-data-plane-adoption-osp-17-to-extracted-crc-minimal-no-ceph
-    parent: cifmw-data-plane-adoption-osp-17-to-extracted-crc
+    parent: adoption-rhel-osp17-standalone-to-crc-no-ceph-provider
+
+- job:
+    name: adoption-rhel-osp17-standalone-to-crc-no-ceph-provider
```

Comment:
so i'm thinking we keep the names we agree on with in https://review.rdoproject.org/r/c/rdo-jobs/+/53788/2/zuul.d/_data_plane_adoption.yaml#4 and then just have a common suffix 

```
-provider
```

to signal that these jobs are in fact content consumers. really 'consumer' is more appropriate but I think some folks would be more familiar with the concept of 'content provider' which is why i used 'provider' here.

WDYT?

@jistr @cescgina @frenzyfriday

Reply:
I'm wondering whether we should explicitly naming the 'provider' ones or the 'no-provider' ones, since i think 'provider' is the usual setup in other repos?

Reply:
so indeed, only data-plane-adoption repo runs this without a content provider.

```
adoption-standalone-to-crc-no-ceph-no-provider
adoption-standalone-to-crc-no-ceph
```

interesting idea and I don't hate it but if we do it then you'd have to "know" that the one without 'no-provider' in the name is the one that has a content provider. 

```
adoption-standalone-to-crc-no-ceph-provider
adoption-standalone-to-crc-no-ceph
```

I am also having second thoughts about not using 'consumer' here... maybe

```
adoption-standalone-to-crc-no-ceph-consumer
adoption-standalone-to-crc-no-ceph
```

I'll update https://review.rdoproject.org/r/c/rdo-jobs/+/53788 today after yesterday's discussion since that one has to go first anyway. then i'll update this and the others that depends-on it and we can finalise this discussion